ppr-ui/src/utils/auth-helper.ts

```diff
@@ -31,3 +38,45 @@ export function getKeycloakRoles (): Array<string> {
   }
   throw new Error('Error getting Keycloak roles')
 }
+
+// Get registering party from auth api /api/v1/orgs/{org_id}
+export async function getRegisteringPartyFromAuth (): Promise<PartyIF> {
+  const url = sessionStorage.getItem('AUTH_API_URL')
+  const currentAccount = sessionStorage.getItem(SessionStorageKeys.CurrentAccount)
+  const accountInfo = JSON.parse(currentAccount)
+  const accountId = accountInfo.id
+
+  const config = { baseURL: url, headers: { Accept: 'application/json' } }
+  return axios.get(`orgs/${accountId}`, config)
+    .then(
+      response => {
+        const data = response?.data
+        if (!data) {
+          throw new Error('Unable to obtain Registering Party from Account Information.')
+        }
+        // Remove null defaults when api address available.
+        const address: AddressIF = {
+          street: data?.contact?.street || 'NA',
+          streetAdditional: data?.contact?.streetAdditional,
+          city: data?.contact?.city || 'NA',
+          region: data?.contact?.region || 'BC',
+          postalCode: data?.contact?.postalCode || 'V8R1V1',
+          country: data?.contact?.country || 'CA',
+          deliveryInstructions: ''
+        }
+        const party: PartyIF = {
+          businessName: data.name,
+          personName: { first: 'Test', last: 'Person' },
```

Comment:
probably want to remove Test Person?

Reply:
Comment in code: // Remove null defaults when api address available.
Defaults are temporary to display address info and pass validation. All address defaults will be removed once the auth api call returns address info.